pkg/products/monitoring/prometheusRules.go

```diff
@@ -292,6 +292,22 @@ func (r *Reconciler) newAlertsReconciler() resources.AlertReconciler {
 					},
 				},
 			},
+			{
+				AlertName: "install-upgrade-alerts",
+				Namespace: r.Config.GetOperatorNamespace(),
+				GroupName: "general.rules",
+				Rules: []monitoringv1.Rule{
+					{
+						Alert: "RHMICSVAbnormal",
+						Annotations: map[string]string{
+							"message": "An abnormal CSV was found, possibly preventing an operator from installing/upgrading",
+						},
+						Expr:   intstr.FromString(fmt.Sprintf("csv_abnormal{reason='RequirementsNotMet',phase='Pending'} or csv_abnormal{reason='RequirementsNotMet',phase='Failed'}")),
+						For:    "15m",
```

Comment:
Leaving this at 15 minutes, adhering to the other warning severity alerts

Reply:
👍